elastic/eui
# fix(datagrid): remove container background

## Summary

This PR removes the background from the `EuiDataGrid` content container.

| Before | After |
| -------|------- |
| ![Screenshot 2024-12-11 at 09 35 11](https://github.com/user-attachments/assets/503142b3-3d44-458f-a632-1e6f273356a5) | ![Screenshot 2024-12-11 at 09 34 49](https://github.com/user-attachments/assets/16f9a04c-55fb-462a-84f2-d1ad931ca58d) |

> [!NOTE]
> Rows are colored separately so changing the container `background-color` to transparent (initial) doesn't affect them.

closes #8136

## QA

### Testing path

1. Run Storybook.
2. Add a story for `EuiDataGrid` with the code from the **reproduction sandbox** (`EuiPanel` wrapping `EuiDataGrid`).
3. Verify that the space underneath the `EuiDataGrid` is transparent.

**Additionally:**

- Check zebra striping.
- Check row highlighting.

### General checklist

- Browser QA
- Code quality checklist
- Release checklist
